public/app/features/explore/Explore.tsx

```diff
@@ -281,6 +284,14 @@ export class Explore extends React.PureComponent<ExploreProps, ExploreState> {
     });
   };
 
+  toggleShowQueryInspector = () => {
```

Comment:
Are we thinking about either:
- closing history tab when inspector is opened and vice versa
- moving one over the other depending on which was open more recently

Reply:
That's one of the remaining questions for sure. On the one hand showing the history drawer closing and the inspector drawer opening (or vice-versa) would be more clear in showing what's happening, but on the other hand I worry that such a sequence of animations would be too much or take too long.
The trouble with just showing, say, the inspector drawer opening over the history drawer is that it's not exactly clear that the history drawer is then closed, so I worry about that a bit

Reply:
Yeah, I agree. I have tested it and I think it looks good as you have implemented it now. Also, I don't think that users will be switching between these two very often.